key.core/src/main/antlr4/KeYParser.g4

```diff
@@ -99,7 +99,7 @@ one_sort_decl
 
 simple_ident_dots
 :
-  simple_ident (DOT simple_ident)* | INT_LITERAL
```

Comment:
For some reason, a number (1,2,3, etc) can also be treated as a dotted identifier (e.g., `java.lang.Object`). 

Hence, `2522` would be a valid name of a sort (`sordId` derives to `simple_ident_dots`). 

This lead to an ambiguity: `23423` can be an `integer`, or and `accessterm`.